sim/core/aura.go

```diff
@@ -49,6 +49,7 @@ type Aura struct {
 	active                     bool
 	activeIndex                int32 // Position of this aura's index in the activeAuras array.
 	onCastCompleteIndex        int32 // Position of this aura's index in the onCastCompleteAuras array.
+	afterCastIndex             int32 // Position of this aura's index in the afterCastAuras array.
```

Comment:
Adding another aura callback is going to slow down everyone's code. This can be done much more easily; just use `OnSpellHitDealt` instead.